_posts/2019-11-04-css-is-hard-and-vanguard-2019-learnings.md

```diff
@@ -0,0 +1,211 @@
+a [standalone](https://www.artsy.net/article/artsy-editorial-rise-fall-internet-art-communities) publication or a
+[part of a larger related series](https://www.artsy.net/series/how-to-be-an-artist) of features. Even less
+frequently, we publish [video](https://www.artsy.net/series/artsy-editors-future-art-carrie-mae-weems) editorials.
+
+To maintain consistency and standardization in terms of UX design, layout, and page interactivity options of the
+editorials that we offer, we use an internal content management service,
+[Artsy Writer](http://pages.artsy.net/rs/609-FDY-207/images/Gallery%20Guide%20to%20Artsy%20Writer.pdf), which
+enables our editorial team to publish content to the main Artsy website without having to collaborate with our
+engineering organization. Yay software! Given that Artsy Writer is designed to handle the layout of the different
+types of editorial features that we offer in a uniform way, highly customized editorials like the
+[2019 Artsy Vanguard](https://www.artsy.net/series/artsy-vanguard-2019) or
+[2018 Year in Visual Culture](https://www.artsy.net/article/artsy-editorial-people-defined-visual-culture-2018)
+require quite a bit of planning and cross-team collaboration, as well as customized software development to support
+the features that deviate from our current layouts.
+
+Creating the 2019 Artsy Vanguard was truly a company-wide effort! I’m pretty new to Artsy, having only joined last
+summer, so some of the work I did on Artsy Vanguard was around understanding our existing framework for routing
+article content from [Positron](https://github.com/artsy/positron), Artsy’s editorial API, to
+[Force](https://github.com/artsy/force), the code that powers our [production site](https://www.artsy.net/). On the
+Engineering side, I collaborated with the project’s technical lead Eve, a senior engineer with years of experience
+working in these codebases. As well as several key stakeholders, including, but certainly not limited to, a lead
+project manager, lead visuals editor, designers, content writers, as well as the
+[external design firm](http://wax-studios.com/) that delivered the initial design concept and video content used
+for the series.
+
+# Learnings
+
+Vanguard was an exciting project, but as with any software endeavor, it came with its own share of learnings and
+difficulties.
+
+## **1. Splitting time between two teams is hard**
+
+My team, Grow, is responsible for the development and maintenance of Engineering software related to increasing
+user engagement with Artsy’s products. Some of our most notable projects include
+[City Guide,](http://files.artsy.net/documents/artsy-bmw-city-guide.pdf) an iOS application built in collaboration
+with BMW, detailing art shows and partner fairs in several major international cities,
+[collections](https://www.artsy.net/collections) and [collection hubs](https://www.artsy.net/collect) offering
```

Comment:
I don't think this is the URL you want to use for collection hubs. The hubs entry point at the top of the collect page is behind an a/b test, so not everyone's going to see it. Should we link to an actual hub page instead?

```suggestion
[collections](https://www.artsy.net/collections) and [collection hubs](https://www.artsy.net/collection/contemporary) offering
```

Reply:
Cool, good thing this sat in a draft mode untouched for a week, we've since launched hubs and now the link works for all users!